src/search/lp/lp_internals.cc

```diff
@@ -145,6 +147,24 @@ unique_ptr<OsiSolverInterface> create_lp_solver(LPSolverType solver_type) {
     }
 }
 
+void set_mip_gap(OsiSolverInterface *lp_solver, double gap) {
+#ifdef COIN_HAS_CPX
+    {
+        auto *cpx_solver = dynamic_cast<OsiCpxSolverInterface *>(lp_solver);
+        assert(cpx_solver);
+        CPXsetdblparam(cpx_solver->getEnvironmentPtr(),
+                       CPXPARAM_MIP_Tolerances_MIPGap, gap);
+    }
+#endif
+#ifdef COIN_HAS_SPX
+    {
+        auto *spx_solver = dynamic_cast<OsiSpxSolverInterface *>(lp_solver);
+        assert(spx_solver);
+        // TODO
```

Comment:
Is this a TODO for this issue? If you want to merge it like this, it probably needs more discussion? Or at least an elaboration of what the TODO is.